app/views/quotes/index.html.erb

```diff
@@ -9,7 +9,7 @@
       </footer>
     </blockquote>
     <div class="created">
-      <%= time_ago_in_words(quote.created_at) %> ago
+      <%= time_ago_in_words(quote.created_at || Time.now) %> ago
```

Comment:
This feels more like a workaround -- lets revert this change and make sure the data can never have a nil timestamp instead. 

We *ought to* experience an error if the timestamp is nil, so I want to leave this code in a way that causes an error if thats the case.